.github/workflows/pr.yml

```diff
@@ -33,6 +33,8 @@ jobs:
       - uses: nrwl/nx-set-shas@v3
       - name: Install & cache node dependencies
         uses: ./.github/actions/install-node-deps
+      - name: '💄 Check format'
+        run: pnpm nx affected --targets=format-check --parallel=2
```

Comment:
Let's run it against the whole repo here, it should not take that much time.
There is no need to leverage Nx here